api_practice/routers/users.py

```diff
@@ -0,0 +1,40 @@
+from fastapi import APIRouter, Depends, status, HTTPException
```

Comment:
we dont need this. We will not offer any user creation services. We will only have a fixed list of users in our code itself.

Reply:
removed the directory and file